faunadb/client.py

```diff
@@ -187,6 +188,30 @@ def query(self, expression, timeout_millis=None):
     """
     return self._execute("POST", "", _wrap(expression), with_txn_time=True, query_timeout_ms=timeout_millis)
 
+  def stream(self, expression, options=None, on_start=None, on_error=None, on_version=None, on_history=None):
+    """
+    Creates a stream Subscription to the result of the given read-only expression. When
+    executed.
+
+    The subscription returned by this method does not issue any requests until
+    the subscription's start method is called. Make sure to
+    subscribe to the events of interest, otherwise the received events are simply
+    ignored.
+
+    :param expression:   A read-only expression.
```

Comment:
Given that what can be streamed at the moment is relatively limited it would be nice to slip in a concrete example of a readonly expresssion that would work into prose of the docstring.

Reply:
A separate PR for documenting streaming functionality would better cater for this. And also tackle the issues raised in #148 
For now, a test case which shows the stream failing with a non-readonly query exists. https://github.com/fauna/faunadb-python/pull/165/files#diff-c7d25fbdfe94b27dca4d863a04f31fd9222cca8b19503976032b95f72f0aa923R57